packages/manager/apps/dedicated/client/app/dedicated/server/details/translations/Messages_fr_FR.json

```
@@ -1271,5 +1271,12 @@
   "server_error_vrack_detach": "Une erreur est survenue lors du détachement du réseau privé : {{error}}",
   "server_error_bandwidth_order": "Une erreur est survenue lors de la commande de bande passante : {{error}}",
   "server_configuration_task_INFRA_002_VIRTUAL_NETWORK_INTERFACE_GROUP": "Interfaces réseau de groupe",
-  "server_configuration_task_INFRA_002_VIRTUAL_NETWORK_INTERFACE_UNGROUP": "Dissocier les interfaces réseau"
+  "server_configuration_task_INFRA_002_VIRTUAL_NETWORK_INTERFACE_UNGROUP": "Dissocier les interfaces réseau",
+  "server_configuration_no_intervention": "Interventions sur site",
+  "server_configuration_on": "On",
+  "server_configuration_off": "Off",
+  "server_configuration_no_intervention_activate_success": "Le Interventions sur site est activé sur le serveur {{serverName}}",
```

```suggestion
  "server_configuration_no_intervention_activate_success": "L'intervention sur site est activée sur le serveur {{serverName}}",
```
Was the content provided ?